src/cloud-api-adaptor/libvirt/config_libvirt.sh

```diff
@@ -123,10 +123,20 @@ echo "libvirt_ssh_key_file=\"id_rsa\"" >> libvirt.properties
 echo "CLUSTER_NAME=\"peer-pods\"" >> libvirt.properties
 
 # switch to the appropriate e2e test and add configs to libvirt.properties as needed
-case $TEST_E2E_SECURE_COMMS in
+case ${TEST_E2E_SECURE_COMMS} in
+
+  withoutKbs)
+    echo "processing withoutKbs"
+    echo "SECURE_COMMS=\"true\"" >> libvirt.properties
+    echo "SECURE_COMMS_NO_TRUSTEE=\"true\"" >> libvirt.properties
+    echo "INITDATA=\"\"" >> libvirt.properties
+    ;;
 
   *)
     echo "processing none"
     echo "SECURE_COMMS=\"false\"" >> libvirt.properties
     ;;
 esac
+
+echo "Content of libvirt.properties"
+cat libvirt.properties
```

Comment:
We already do this in the CI (https://github.com/confidential-containers/cloud-api-adaptor/blob/1079abfd3e430a08bc657bb85b565a1b0b8be832/.github/workflows/e2e_libvirt.yaml#L123), so it's not needed here